test/acceptance/docker-compose.yaml

```diff
@@ -9,4 +9,9 @@ services:
       CHROME_ARGS: "--headless --ignore-certificate-errors"
       BROWSERSTACK_USERNAME: ${BROWSERSTACK_USERNAME}
       BROWSERSTACK_ACCESS_KEY: ${BROWSERSTACK_ACCESS_KEY}
-      BROWSERSTACK_PARALLEL_RUNS: ${BROWSERSTACK_PARALLEL_RUNS}
\ No newline at end of file
+      BROWSERSTACK_PARALLEL_RUNS: ${BROWSERSTACK_PARALLEL_RUNS}
+      JWT_SECRET: ${JWT_SECRET}
```

Comment:
Can we rename these so it is clear they are for Notify? 

Separate conversation on how we get them in Jenkins for example -> 

https://github.com/DEFRA/ffc-jenkins-pipeline-library/blob/f1f75434faa46ba747731e7d10a628fecb2a78dd/src/uk/gov/defra/ffc/Tests.groovy#L204